.github/workflows/release.yml

```diff
@@ -168,34 +168,27 @@ jobs:
       - uses: denoland/setup-deno@v1
         with:
           deno-version: ${{ env.DENO_VERSION }}
-      - uses: metatypedev/setup-ghjk@2e8bbf084060a18828338a7cdd43fde6feb2a3cc
+      - uses: metatypedev/setup-ghjk@318209a9d215f70716a4ac89dbeb9653a2deb8bc
       - shell: bash
         env:
           POETRY_PYPI_TOKEN_PYPI: ${{ secrets.PYPI_TOKEN }}
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
           WASM_FILE: target/release/typegraph_core.wasm
         run: |
-          cargo build -p typegraph_core --target wasm32-unknown-unknown --target-dir target/wasm --release
-          mkdir -p $(dirname $WASM_FILE)
-          wasm-opt -Oz target/wasm/wasm32-unknown-unknown/release/typegraph_core.wasm -o $WASM_FILE.opt
-          wasm-tools component new $WASM_FILE.opt -o $WASM_FILE
+          ghjk x install-py
+          ghjk x install-ts
+
+          WASM_OPT=1 ghjk x build-tgraph-core
```

Comment:
should we allow user to define flag and arg `ghjk x task --opt`? that would be great only if there is an automated doc and help generated

Reply:
Yeah, good point. The CLI library we're using currently has some serious [limitations](https://github.com/c4spar/deno-cliffy/discussions/696) and I was thinking I'd first move to either `zcli` or `clap` before implementing that.